common/src/test/java/com/sequenceiq/cloudbreak/common/anonymizer/AnonymizerUtilTest.java

```diff
@@ -28,6 +28,72 @@ public void testCapitalHidePasswordSingleQuote() {
                 anonymize(testData));
     }
 
+    @Test
+    public void testWithBothSimpleAndDoubleQuotes() {
+        String testData = ""
+                + "export test1_password=pwd\n"
+                + "export test1_pass=pwd\n"
+                + "export test1_secret=pwd\n"
+                + "export test1_key=pwd\n"
+                + "export test1_credential=pwd\n"
+
+                + "export test2_password:pwd\n"
+                + "export test2_pass:pwd\n"
+                + "export test2_secret:pwd\n"
+                + "export test2_key:pwd\n"
+                + "export test1_credential:pwd\n"
+
+                + "export test3_password:'pwd'\n"
+                + "export test3_pass:'pwd'\n"
+                + "export test3_secret:'pwd'\n"
+                + "export test3_key:'pwd'\n"
+                + "export test3_credential:'pwd'\n"
+
+                + "export test4_password=\"pwd\"\n"
+                + "export test4_pass=\"pwd\"\n"
+                + "export test4_secret=\"pwd\"\n"
+                + "export test4_key=\"pwd\"\n"
+                + "export test4_credential=\"pwd\"\n"
+
+                + "export \"test5_password\":\"pwd\"\n"
+                + "export \"test5_pass\":\"pwd\"\n"
+                + "export \"test5_secret\":\"pwd\"\n"
+                + "export \"test5_key\":\"pwd\"\n"
+                + "export \"test5_credential\":\"pwd\"\n";
+
```

Comment:
Can you please add a test for `password=\'pwd\'` `pass=\'pwd\'` etc.? I think validation is missing for all the cases with an equals symbol and with single quotes.

Reply:
Sure!

Reply:
test3 is with single quotes but with `:`. I can add tests with `=` and single quote.